benchmarks/benchmark_registration.py

```diff
@@ -1,6 +1,11 @@
-from skimage.data import stereo_motorcycle
 from skimage.color import rgb2gray
-from skimage import registration
+
+try:
+    from skimage import registration
+    from skimage.data import stereo_motorcycle
+    have_registration = True
+except ImportError:
+    have_registration = False
```

Comment:
Do you need to do something different below for this to matter?

Reply:
There is the following below to raise a more informative error when the module doesn't exist.

```Python
        if not have_metrics_module:
            return NotImplementedError("metrics module unavailable")
```

Reply:
Actually on further review, this `NotImplementedError` should be raised within the `setup` method and not the test method to to get the test to show up as `n/a` instead of `failed`

e.g.

```
[ 50.00%] ··· benchmark_registration.RegistrationSuite.time_tvl1                 n/a
```